README.md

```diff
@@ -152,39 +152,39 @@ shown below. Can't see yours? Have a look at the [extra platform instructions](h
 
```

Comment:
The same comment as before applies to the rest of the headings in this file.
Please check the rich-diff to confirm that this only changes formatting, not the output.